meta/open_standup.md

```diff
@@ -14,10 +14,11 @@ sharing; they are an organization-wide responsibility and every engineer should
 
 * The person handling the talking parts `@here`s the dev channel. Include where the meeting is in the NYC office
   (usually the Classroom).
+* The same person reminds last week's on-call staff in #dev to prepare their updates about last week's rotation. Something like:
+  > @personA @personB reminder, we're looking for a list of major or notable incidents from last week during standup.
```

Comment:
This looks good! Can we also add that the meeting leader should announce who the new on-call people are for the coming week + the week after during the standup? (I think it's good for people to get a heads up a little in advance). This info is available on the on-call calendar!